cpp/src/Glacier2Lib/SessionHelper.cpp

```diff
@@ -450,20 +450,22 @@ SessionHelperI::connected(const Glacier2::RouterPrx& router, const optional<Glac
     //
     Ice::ConnectionPtr conn = router->ice_getCachedConnection();
     string category = router->getCategoryForClient();
-    int32_t acmTimeout = 0;
+    int32_t remoteIdleTimeout = 0;
     try
     {
-        acmTimeout = router->getACMTimeout();
+        remoteIdleTimeout = router->getACMTimeout();
```

Comment:
Do we plan on renaming/replacing this function `getACMTimeout`?

Reply:
These are unfortunately Slice-defined operations. As a result, we can't rename them without breaking on-the-wire compatibility. 

For Glacier2, we want to keep on the wire compatibility in both directions:
 - an Ice 3.7 client using an Ice 3.8 server
 - an Ice 3.8 client using an Ice 3.7 server

For IceGrid, I am not sure. Do we want to support deployments with a mix of IceGrid 3.7 and 3.8 for the registry replicas and nodes? If yes, it gets complicated given the way "keep alive" are implemented in IceGrid.